.github/workflows/observer-cd.yml

```diff
@@ -0,0 +1,45 @@
+name: Observer CD
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - staging
+      - trying
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+
+jobs:
+  docker-push:
+    name: Push to container registry
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.17'
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Build relic
+      run: make crypto_setup_gopath
+    # Provide Google Service Account credentials to Github Action, allowing interaction with the Google Container Registry
+    # Logging in as github-actions@dl-flow.iam.gserviceaccount.com
+    #- name: Docker login
+    #  uses: docker/login-action@v1
+    #  with:
+    #    registry: gcr.io
+    #    username: _json_key
+    #    password: ${{ secrets.GCR_SERVICE_KEY }}
+    - name: Docker build
```

Comment:
why is this commented out?

Reply:
Thank you for the review.
The final implementation used environment variables set up for the GitHub actions, instead of a login.
The original instructions we received targeted an Observer Microservice in a different repository.
Since we are in the flow-go repository still, we just ignore this file for the PR.
The feedback request was valuable in any case.